packages/website/src/components/linter/createLinter.ts

```diff
@@ -160,7 +167,37 @@ export function createLinter(
       console.log('[Editor] Updating', fileName, compilerOptions);
       parser.updateConfig(compilerOptions);
     } catch (e) {
-      console.error(e);
+      if (e instanceof Error) {
+        const error = {
+          group: 'Typescript',
+          items: e.message
+            .trim()
+            .split('\n')
+            .map((message: string) => {
+              return {
+                fixer: undefined,
+                location: '',
+                message,
+                severity: 8,
```

Comment:
[Docs] It's not clear where this magic number comes from at first. Keeping in line with... https://github.com/typescript-eslint/typescript-eslint/blob/ee38b525449513ce56632aee4a7a8d40871672ee/packages/website/src/components/linter/utils.ts#L144-L146

```suggestion
                severity: 8, // MarkerSeverity.Error
```